### PR TITLE
Adding imdclient package of IMDv3

### PR DESCRIPTION
**Pull request by hcho38**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by orbeckst** [comment]
@hcho38 can you please add me to the maintainer list? Thanks.

I am willing to be a maintainer.

**Comment by ljwoods2** [comment]
I am willing to be a maintainer

File: recipes/imdclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    IMDv3, the protocol version described in this document, builds upon IMDv2, which is implemented at the time of writing in NAMD, VMD, GROMACS, and LAMMPS.
+    IMDv2 is itself a modification of the original IMD protocol published in 2001.
+  doc_url: https://imdclient.readthedocs.io/en/latest/protocol_v3.html
+  dev_url: https://github.com/Becksteinlab/imdclient
+
+extra:
+  recipe-maintainers:
+    - hcho38
+    - ljwoods2
+    - amruthesht
```
Comment (orbeckst) [inline recipes/imdclient/meta.yaml:53]: I am willing to be a maintainer (and the package is hosted in my groups GitHub so I have to ensure that at least one person has long-term access).
```suggestion
    - amruthesht
    - orbeckst
```

**Comment by amruthesht** [comment]
I am willing to be a maintainer

**Comment by hcho38** [comment]
@conda-forge/help-python ready for review

**Review by synapticarbors — CHANGES_REQUESTED** [review]
Overall looks good, but could you have a look at our docs on `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

We are now enforcing a `python_min`, so you'll want to follow the convention we have there.

**Comment by hcho38** [comment]
@synapticarbors I made the changes to the recipe based on your comment and the documentation

File: recipes/imdclient/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=61.2
+    - versioningit
+    - pip
+  run:
+    - python {{ python_min }}
```
Comment (synapticarbors) [inline recipes/imdclient/meta.yaml:24]:
```suggestion
    - python >={{ python_min }}
```

File: recipes/imdclient/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - mdanalysis >=2.7.0
+
+test:
+  imports:
+    - imdclient
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (synapticarbors) [inline recipes/imdclient/meta.yaml:33]:
```suggestion
    - pip
    - python {{ python_min }}
```

**Comment by orbeckst** [comment]
Thank you, @synapticarbors !

**Comment by synapticarbors** [comment]
@orbeckst you're welcome! It's always fun to see biophysics folks from my previous life pop up on here.

**Pull request merged**